aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+.. _web-api-help-pages-using-swagger:
+
+ASP.NET Web API Help Pages using Swagger
+========================================
+
+By `Shayne Boyer`_
+
+Understanding the various methods of an API can be a challenge for a developer when building a consuming application. 
+
+Generating good documentation and help pages as a part of your Web API using `Swagger <a href="http://swagger.io/">`_ with the .NET Core implementation `Swashbuckle <a href="https://github.com/domaindrivendev/Ahoy">`_ is as easy as adding a couple of NuGet packages and modifying the Startup.cs.
+
+This tutorial builds on the sample on :doc:`first-web-api`. If you'd like to follow along, download the sample at https://github.com/aspnet/Docs/tree/master/aspnet/tutorials/first-web-api/sample.
+
+.. contents:: Sections:
+  :local:
+  :depth: 2
+
+Getting Started
+---------------
+There are two core components to Swashbuckle 
+
+- *Swashbuckle.SwaggerGen* : provides the functionality to scaffold your Web API and generate JSON Swagger documents that describe the objects, methods, return types, etc. 
+- *Swashbuckle.SwaggerUI* : an embedded version of the Swagger UI tool which uses the above documents for a rich customizable experience for describing the Web API functionality and includes built in test harness capabilities for the public methods.
+
+NuGet Packages
+--------------
+You can add Swashbuckle with any of the following approaches:
+
+- From the Package Manager Console: 
+
+.. code-block:: bash
+
+    Install=Package Swashbuckle -Pre
+
+- Add Swashbuckle to project.json:
+
+.. code-block:: javascript
+
+    "Swashbuckle": "6.0.0-"
```

Comment:
@Rick-Anderson - the first PR review you stated that they should all be literal includes????